src/app/legacy/containers/BulletedListItem/index.jsx

```diff
@@ -64,6 +71,7 @@ export const ParagraphPropTypes = {
   ).isRequired,
 };
 
+// causing a console error since it's expecting a paragraph
```

Comment:
Is it possible to fix this? (If not, I wouldn't worry about adding this comment, I am planning to clean up / suppress errors in https://github.com/bbc/simorgh/pull/10541)

Reply:
Oops yh that was a note to self. I took a look at fixing this. I know the cause of this is that I'm using these legacy containers to render the 'post' data, which is in a slightly different format to the 'article' data. But I wan't able to get to the bottom of how to fix the errors. (Evidence why Typescript makes life waay easier.)

I believe the plan is to a complete refactor the way the nested paragraph containers work (similar to how Aaron did a [complete overhaul of images](https://github.com/bbc/simorgh/pull/11056)). So my approach is potentially a "short term" solution. (Fully appreciating that short term could still be a long time)

So all this said, I'm not sure how long to spend trying to find and fix the cause of the errors. If your PR is a good way to solve the problem of the console errors, then that would be great.